include/system/analytics_manager.js

```diff
@@ -112,14 +128,16 @@ module.exports = function AnalyticsManagerModule(pb) {
      * instance of Localization.  The last is a callback that should be called with 
      * two parameters.  The first is an error, if occurred and the second is raw 
      * HTML string that represents the snippet to be executed by the analytics 
-     * plugin. 
+     * plugin.
+     * @param {String} optional name of template tag where the template service to register your snippet 
```

Comment:
Would you mind updating the documentation to:

```
* @param {String} name
* @param {Function} onPageRendering 
* @param {String} [templateTag='analytics'] name of template tag where the template service to register your snippet
```

This will make things show up correctly when we generate our documentation using YUIDoc